src/main/java/org/folio/helper/PurchaseOrderLineHelper.java

```diff
@@ -441,18 +441,10 @@ private boolean isEncumbranceUpdateNeeded(CompositePurchaseOrder compOrder, Comp
       return true;
     }
 
-    return requestFundDistros.stream()
-      .map(reqFd -> storageFundDistros.stream().anyMatch(storageFd -> fundDistributionChanged(reqFd, storageFd).test(false)))
-      .anyMatch(isAnyFdChanged -> isAnyFdChanged.equals(Boolean.TRUE));
+    return !CollectionUtils.containsAll(requestFundDistros, storageFundDistros)
+      || !CollectionUtils.containsAll(storageFundDistros, requestFundDistros);
```

Comment:
I believe it is possible to replace it by `!isEqualCollection(storageFundDistros, requestFundDistros)`

Reply:
Sure. I was thinking it will not work for the same collections, but with different order of elements. But it works